.github/workflows/deploy.yml

```
@@ -246,6 +247,17 @@ jobs:
           aws s3 cp ${{ env.CORE_ARTIFACTS_BASE_PATH }}/legacy/iife/ $s3_path_prefix/legacy/ $copy_args
           aws s3 cp ${{ env.CORE_ARTIFACTS_BASE_PATH }}/modern/iife/ $s3_path_prefix/modern/ $copy_args
 
+      # The loader sits at the root of the version directory rather than under
+      # legacy/ or modern/ because its job is to detect which of the two to fetch.
+      - name: Copy SDK loader artifacts to S3 (versioned directory)
+        if: ${{ inputs.environment == 'production' }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:4:85: Double quote to prevent globbing and word splitting

(shellcheck)